lib/src/main/java/uk/gov/di/ipv/core/library/service/ConfigurationService.java

```diff
@@ -229,4 +254,55 @@ public String getCoreVtmClaim() {
         return ssmProvider.get(
                 String.format("/%s/core/self/coreVtmClaim", System.getenv(ENVIRONMENT)));
     }
+
+    public String getCriPrivateApiKey(String criId) {
+        GetSecretValueRequest valueRequest =
+                GetSecretValueRequest.builder()
+                        .secretId(
+                                String.format(
+                                        "%s/credential-issuers/%s/api-key",
+                                        System.getenv(ENVIRONMENT), criId))
+                        .build();
+
+        try {
+            String secretManagerKeyValueJson = getSecretsManagerValue(valueRequest);
+
+            if (secretManagerKeyValueJson != null) {
+                Map<String, String> secret =
+                        objectMapper.readValue(secretManagerKeyValueJson, new TypeReference<>() {});
+                return secret.get(API_KEY);
+            }
+            return null;
```

Comment:
Generally we should prefer to use `Optional` rather than returning `null`s so callers of this function must consider the null case and can do so in a "nicer" way, but I think refactoring to do that now would mean changing quite a bit in this PR so in the interest of time and that this works lets move forward. Something to consider for next time if you like 👍